### PR TITLE
allow passing force=true in delete calls

### PR DESCRIPTION
**Pull request by wheresrhys**
## Why?

The API now allows sending force=true to delete requests
## What?
- adds a params object to `delete`
- documents and tests it

### CODE REVIEW

File: lib/api/node.js [hunk 1]
```diff
@@ -71,12 +71,15 @@ function buildNode(options) {
 	 * Make a Biz Ops API DELETE request to destroy an existing record
 	 * @param {String} type
 	 * @param {String} code
+	 * @param {Object} [params={}] - additional query string parameters
+	 * @param {Boolean} [params.force] - allow the deletion of a node even if it has relationships
```
Comment (GeoffThorpeFT) [inline lib/api/node.js:75]: Is the explicit reference to a param for the boolean valid?  Its existence has already been stated by the object parameter. The function does only accept three parameters; not the four suggested by the inclusion of the boolean.

Comment (wheresrhys) [reply lib/api/node.js:75]: It's the same level of detail as all the other methods (I copied what Matt did there) so I presume it's correct

**Review by i-like-robots — APPROVED** [review]
Fab

File: lib/api/node.js (hunk 1 above)
Comment (i-like-robots) [reply lib/api/node.js:75]: Yep, it's all good: https://jsdoc.app/tags-param.html#parameters-with-properties

**Pull request merged**